src/charm.py

```diff
@@ -85,7 +90,12 @@ def reconcile(self) -> None:
             self.certificates.private_key,
             self.state.certificate,
         )
+        if self.state.git_repository:
+            wazuh.configure_git(container, str(self.state.git_repository), self.state.git_ssh_key)
```

Comment:
What is the reason for needing to convert to `str` here please?

Reply:
Actually, I don't need it. Removed